### PR TITLE
Use daiquiri logging for log messages

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
There will be a merge conflict with #29, I will rebase once needed.

File: runtimes/Dockerfile-ml [hunk 1]
```diff
@@ -14,7 +14,8 @@ RUN pip install \
     numpy \
     pandas \
     scipy \
+    daiquiri \
```
Comment (luebken) [inline runtimes/Dockerfile-ml]: Note that we are changing the basic runtime here. Which is fine if daiquiri is the way to go. Just double checking.

**Comment by fridex** [comment]
Rebased.

File: runtimes/Dockerfile-ml (hunk 1 above)
Comment (fridex) [reply runtimes/Dockerfile-ml]: If the image should be generic enough ML image, it is probably not a good idea to include daiquiri in that case.

**Pull request closed without merging**